actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/OrgBulkUploadActor.java

```diff
@@ -78,6 +77,12 @@ private void upload(Request request) throws IOException {
               .collect(
                   Collectors.toMap(
                       entry -> (entry.getKey()).toLowerCase(), entry -> entry.getValue()));
+      Map<String, Object> lowerCaseInternalNamesMap = new HashMap<>();
+      supportedColumnsMap.forEach(
+          (String k, Object v) -> {
+            lowerCaseInternalNamesMap.put(v.toString().toLowerCase(), v.toString());
```

Comment:
Line 79 -> it is putting lower-case on key - value(original Case)
Line 83 -> it is putting lower-case on value- value(original Case)

So if config is like : {"SCHOOL NAME": "orgName"}
output would be -> {"school name": "orgName", "orgname":"orgName"}

So that look-up as well as validation of column works correctly.

Reply:
OK. Can we directly update supportedColumnsList instead of creating another map lowerCaseInternalNamesMap?

Reply:
No, because this is not just used for validation, but also for look-up purpose, to get internal name. 
So if we just add to supported columns list directly, validation 1 - supported column would be ok. 
But next validation mandatory columns -> which is just based on reverse look-up would fail.

like -> orgName is mandatory -> lowerCase lookup -> {"school name": "orgName"}
Passed value: "orgname" -> hence it will throw mandatory value not found.